src/draco/tools/fuzz/draco_databuffer_fuzzer.cc

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "draco/src/draco/core/data_buffer.h"
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+  draco::DataBuffer buffer;
+  buffer.Update(reinterpret_cast<const void *>(data), static_cast<int64_t>(size));
```

Comment:
I'm not sure this does enough to warrant a full fuzzer. It only resizes and copies data. But having it shouldn't cause any harm.